rootfs/etc/nginx/lua/configuration.lua

```diff
@@ -221,9 +221,9 @@ local function handle_backends()
 end
 
 function _M.call()
-  if ngx.var.request_method ~= "POST" and ngx.var.request_method ~= "GET" then
+  if ngx.var.request_method ~= "POST" then
     ngx.status = ngx.HTTP_BAD_REQUEST
-    ngx.print("Only POST and GET requests are allowed!")
+    ngx.print("Only POST requests are allowed!")
```

Comment:
It seems the following code snippet can be removed, since the method is always `POST`.

```lua
if ngx.var.uri == "/configuration/certs" then
    handle_certs()
    return
  end
```

Reply:
And the `local function handle_certs()` can be removed too.
Anyway, we can do the clean work later.

Reply:
Yes, I prefer leaving this for a followup clean up, sounds good?